### DISCUSSION TITLE
Does harp.gl support OGC 3d tiles?

### DISCUSSION THREAD

**Discussion by LukeParky** [heremaps/harp.gl#2235] [Q&A] [answered]
Hi there, I am struggling to find an answer to this in the documentation. Can custom 3D building data be supplied to a harp.gl map in [OGC 3D tile format](http://docs.opengeospatial.org/cs/18-053r2/18-053r2.html)? Or are all 3D building height data supplied through the Here data sources?  Thank you

**Comment by nzjony** [comment, marked as answer]
hi @LukeParky , sorry for the delay, I was on holiday. Cool to see another Kiwi on Github, I come from NZ, but now live in Berlin.

We don't currently support this format. I don't know of any plans to support this format, but if there is some demand, for sure we could consider it.

The buildings are extruded, i.e. it is some polygon with some height value via either via the HERE Data source, or this can also be done via GeoJSON

If you have buildings in the GLTF format, you can easily load them and add them to the map using the three.js interface.

Hope that helps answer your question.
